src/main/java/org/fogbowcloud/saps/engine/core/dispatcher/restlet/resource/BaseResource.java

```diff
@@ -33,7 +33,7 @@ protected boolean authenticateUser(String userEmail, String userPass, boolean mu
 
 		SapsUser user = application.getUser(userEmail);
 		String md5Pass = DigestUtils.md5Hex(userPass);
-		if (user != null && user.getUserPassword().equals(md5Pass) && user.getActive()) {
+		if (user != null && user.getUserPassword().equals(md5Pass) && user.getEnable()) {
```

Comment:
this name is not ok. I think you `isEnabled`?

Reply:
`isEnabled` gets better, I changed it.